src/benchmark/benchmark_join_fixture.cpp

```diff
@@ -1,127 +0,0 @@
-#include "benchmark_join_fixture.hpp"
-
-#include <memory>
-#include <vector>
-
-#include "benchmark/benchmark.h"
-#include "operators/table_wrapper.hpp"
-#include "storage/chunk.hpp"
-#include "storage/index/adaptive_radix_tree/adaptive_radix_tree_index.hpp"
-#include "storage/storage_manager.hpp"
-#include "table_generator.hpp"
-#include "types.hpp"
-
-namespace opossum {
-
-void BenchmarkJoinFixture::SetUp(::benchmark::State& state) {
```

Comment:
I can see that you don't want to have all the benchmark being part of the speed center, but does this removal delete all the benchmark code as well?

Reply:
Yes, kind of. Although the benchmark itself did not get removed, I just removed the many configurations. I simplified the setup a little during that (e.g. getting rid of the join fixture), because it is not necessary, and is more readable that way, both in code as well in the output.

Reply:
I absolutely don't like to throw away valid code that was created with an intention that still makes sense and needs to be covered.

I am totally fine with removing it from regular speed center runs, but not from removing it completely.

Or do I miss something?

Reply:
Without having looked into @lanice changes yet, I'd like to weigh in that our exisiting benchmark  suite was quite a chaos (very long execution time, benchmarking cross products, etc.) and cleaning it up makes sense.

Reply:
And to say again, there really is not a lot that got thrown away (okay, the product benchmark actually got thrown away..). In the JoinBenchmark for example it looks like there was deleted a lot, but really I just moved the implementation into one file and reduced a lot of boilerplate code. If you want to execute it with the previous configurations, you only have to adjust a) the `generate_table` call here:
```
template <class C>
void BM_Join(benchmark::State& state) {
  auto table_wrapper_left = generate_table(100000, 50);
  auto table_wrapper_right = generate_table(100000, 50);

  BM_Join_impl<C>(state, table_wrapper_left, table_wrapper_right);
}
```
and (if wanted) the `ColumnDataDistribution` in the `generate_table` function.